tokens/style-dictionary.js

```diff
@@ -296,15 +406,28 @@ const initializeStyleDictionary = async ({ themes }) => {
    * @param {StyleDictionaryFilterOptions} [filterOptions] Custom options for the filter.
    */
   function registerStyleDictionaryFilter(name, filter, filterOptions = {}) {
-    StyleDictionary.registerFilter({ name, filter });
+    StyleDictionary.registerFilter({
+      name,
+      filter: (token, opts) => {
+        const paragonExtensions = token.$extensions?.[paragonExtensionsNamespace];
+        const isReferencedToken = paragonExtensions?.isReferencedToken;
+        const baseFilterResult = typeof filter === 'function' ? filter(token, opts) : filter;
+        return baseFilterResult || isReferencedToken;
```

Comment:
Each custom registered filter will rely on its own `filter` definition _OR_ fallback to whether the token is a reference token (i.e., another token uses it in its `$value` as an alias). Same for the `themeVariant` specific version of the custom filter.

This change makes it possible to include CSS variable definitions in output files that are otherwise not defined, despite previously intentionally being excluded due to being assumed to exist in other output files. Doing so removes the somewhat alarming warning messages that previously were emitted about having output references without their definitions in output files. With this change, these warnings go away in both Paragon and brand packages.